yarn-project/end-to-end/src/e2e_fees.test.ts

```diff
@@ -280,6 +282,27 @@ describe('e2e_fees', () => {
         })
         .wait();
 
+      /**
+       * at present the user is paying DA gas for:
+       * 3 nullifiers = 3 * DA_BYTES_PER_FIELD * DA_GAS_PER_BYTE = 3 * 32 * 16 = 1536 DA gas
+       * 2 note hashes =  2 * DA_BYTES_PER_FIELD * DA_GAS_PER_BYTE = 2 * 32 * 16 = 1024 DA gas
+       * 964 bytes of logs = 964 * DA_GAS_PER_BYTE = 964 * 16 = 15424 DA gas
+       * tx overhead of 512 DA gas
+       * for a total of 18496 DA gas.
+       *
+       * The default teardown gas allocation at present is
+       * 100_000_000 for both DA and L2 gas.
+       *
+       * That produces a grand total of 200018496n.
+       *
+       * This will change because:
+       * 1. Gas use during public execution is not currently incorporated
+       * 2. We are presently squashing notes/nullifiers across non/revertible during private exeuction,
+       *    but we shouldn't.
+       */
+
+      expect(tx.transactionFee).toEqual(200018496n);
```

Comment:
I'd suggest we actually compute this value here in code, using the constants, so it's easier to update if we change any of the inputs